hermes/file_gen.go

```diff
@@ -0,0 +1,33 @@
+package hermes
+
+import (
+	"fmt"
+	"crypto/sha1"
+)
+
+type HermesFile struct {
+	name string
```

Comment:
Should `Name` and `Contents` be exported?

Reply:
```suggestion
	Name string
```